clients/drcachesim/common/options.cpp

```diff
@@ -639,12 +652,14 @@ droption_t<bytesize_t> op_L0_filter_until_instrs(
     "full trace. Therefore TRACE_MARKER_TYPE_WINDOW_ID markers indicate start of "
     "filtered records.");
 
+// XXX i#7230: Currently the simulators count non-marker records here: we should
```

Comment:
s/count non-marker/count only non-marker/

Reply:
Done, and in the other instance.